docs/source/resources/compiler-tools.rst

```diff
@@ -317,7 +317,7 @@ With that warning in mind, let's look at good ways to customize clang.
    Activate scripts are named according to our package name so they won't
    conflict with other activate scripts.
 
-   The symlink for clang is a clang implementation detail that sets the host
+   The symlink for Clang is a Clang implementation detail that sets the host
```

Comment:
ambivalent on caps here.  I lean towards lowercase, but not strongly.

Reply:
Their documentation capitalizes it.